LDMP/data_io.py

```diff
@@ -1094,8 +1138,10 @@ def warp_raster(self, out_file):
         log("Importing {} to {}".format(in_file, out_file))
 
         if self.input_widget.groupBox_output_resolution.isChecked():
-            out_res = self.get_out_res_wgs84()
-            resample_mode = self.get_resample_mode(temp_vrt)
+            out_res = None  # self.get_out_res_wgs84()
+            resample_mode = (
+                gdal.GRA_NearestNeighbour
```

Comment:
This changes the logic - in past versions of T.E. `get_resample_mode` was used to use either `gdal.GRA_Mode` for resampling categorical data or `gdal.GRA_Average` for continuous data when aggregating pixels to go from higher to lower resolution. Better to retain that login here rather than always use `gdal.GRA_NearestNeighbour`